src/main/java/com/greenfoxacademy/goddesstribesbackend/models/entities/Building.java

```diff
@@ -19,6 +19,7 @@
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   private Long id;
 
+  private BuildingTypeENUM buildingType;
```

Comment:
I think `type` is enough, because it's a building, it is magatolertetodo

Reply:
Barni, this is also an added extra file, not related to the TGB7, written by Judit. Since you or Bendegúz approved it previously do you mind if it remains like this?